functions/package.json

```diff
@@ -1,13 +1,18 @@
 {
-  "name": "functions",
-  "description": "Cloud Functions for Firebase",
+  "name": "bss-firebase-functions",
+  "version": "0.1.0",
+  "description": "BedStuyStrong Cloud Functions for Firebase",
   "scripts": {
-    "lint": "eslint .",
-    "serve": "firebase emulators:start --only functions",
-    "shell": "firebase functions:shell",
+    "config:get": "npx firebase functions:config:get",
+    "config:test": "npx firebase use test",
+    "deploy": "npx firebase deploy",
+    "lint": "npx eslint .",
+    "lint:fix": "npx eslint . --fix",
+    "logs": "npx firebase functions:log",
+    "postversion": "git push --all && git push --tags",
+    "serve": "npx firebase emulators:start --only functions",
+    "shell": "npx firebase functions:shell",
```

Comment:
- some overlap with #34 but they sync and only because i "co-developed" 😄 
- `npx` things to rely on `devDependencies`